src/sites/image/reklama.js

```diff
@@ -272,8 +272,35 @@
     ready: defaultAction,
   });
 
+  $.register({
+    rule: {
+      host: /^imagerar\.com$/,
+      path: /^\/img-/
+    },
+    ready: function () {
+      $.openLink(linkid, {replace: true});
```

Comment:
You should access `linkid` via `$.window`.
Also, `$.openLink` does not have `replace` option.

Please merge this rule into https://github.com/adsbypasser/adsbypasser/blob/c7d49b6d3acffbc08c4edadf9296704e45ec53bc/src/sites/image/reklama.js#L251